src/tooling/simulating/qemu.md

```diff
@@ -15,18 +15,20 @@ Once you have built QEMU, you should have the `qemu-system-xtensa` file.
 For running our project in QEMU, we need a firmware/image with bootloader and partition table merged in it.
 We can use [`cargo-espflash`][cargo-espflash] to generate it:
 
-```shell
+```console
```

Comment:
Is there any reason to switch code snippets to `console` from `shell`. The rest of the book uses `shell` so it might be worth to keep it consistent.

Reply:
OK, back to using bash here.
I may send another PR to use ShellSession.